spec/forms/search_keywords_form_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SearchKeywordsForm, type: :form do
+  describe '#seach_keywords' do
+    context 'given user does not have keyword' do
+      it 'returns an empty array' do
+        Fabricate(:keyword)
+        user = Fabricate(:user)
+        filter_params = { adwords_url_contains: 'vpn' }
+        pagination_params = { page: 1, items: 2 }
+        _pagination, keywords = described_class.new(user.keywords, filter_params, pagination_params).search_keywords
+
+        expect(keywords).to be_empty
+      end
+    end
+
+    context 'when user has keyword' do
```

Comment:
```suggestion
    context 'given a user has keywords' do
```

Or you need to update the above context at line `7` to `when...` I am fine with both but just keep it consistent.